db/version_edit.cc

```diff
@@ -120,6 +120,9 @@ bool VersionEdit::EncodeTo(std::string* dst) const {
   if (has_max_column_family_) {
     PutVarint32Varint32(dst, kMaxColumnFamily, max_column_family_);
   }
+  if (has_min_log_number_to_keep_) {
+    PutVarint32Varint64(dst, kMinLogNumberToKeep, min_log_number_to_keep_);
+  }
```

Comment:
Just wondering if this is safe... IIUC the reason this wasn't encoded earlier had to do with the MANIFEST not being forward compatible (see the `kMinLogNumberToKeepHack` custom tag of `kNewFile4`)

Reply:
Right, but as early as 5.14, RocksDB is able to understand `kMinLogNumberTokeep` (https://github.com/facebook/rocksdb/blob/5.14.fb/db/version_edit.cc#L360), so I guess it's OK? It means user cannot use earlier versions than 5.14 to open a DB generated with versions newer than 7.0 (if we backport).